force-app/main/default/classes/SYS_CustomException.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>45.0</apiVersion>
+    <status>Active</status>
+</ApexClass>
```

Comment:
Same here.

Reply:
This is the metafile of the SYS_CustomException.cls